ansible/files/telemetry.sh

```diff
@@ -85,6 +85,9 @@ echo "export PYSPARK_SUBMIT_ARGS=\"--packages com.databricks:spark-csv_2.10:1.2.
 
 source $HOME/.bashrc
 
+# Setup Jupyter notebook
+aws s3 cp $TELEMETRY_CONF_BUCKET/bootstrap/jupyter_notebook_config.py ~/.jupyter/jupyter_notebook_config.py
```

Comment:
I'm OK with storing the file in S3 directly, but we need to include some kind of automation to create it in the first place (even if it's just documentation that says where to get the clientid and client secret).